inc/vcf/assembly_report_writer.hpp

```diff
@@ -129,6 +134,11 @@ namespace ebi
             file << vcf_line;
         }
 
+        virtual void write_meta_info(std::string &meta_info_line) override
```

Comment:
Now that this code is in place, remove the `TODO` at the beginning of this class.